.github/CODEOWNERS

```diff
@@ -41,6 +46,11 @@
 # PRLabel: %Storage
 /sdk/storage/            @vinjiang @Jinming-Hu @EmmaZhu @antkmsft @gearama @LarryOsterman @microzchang
 
+# AzureSDKOwners: @gearama
+# ServiceLabel: %Tables
+# PRLabel: %Tables
+/sdk/tables/            @gearama @LarryOsterman
```

Comment:
Should we add some more people on this line? if it is used when starting PRs, it would be nice to not have to add people manually.

Reply:
Fixed